Isle of Trust/src/components/EndOfLevel.tsx

```diff
@@ -78,5 +78,31 @@ export function EndOfLevel({ level, success, mapAgents }: EndOfLevelProps) {
     );
 }
 
+interface OutOfResourcesProps {
+    level: number;
+};
+export function OutOfResources({ level }: OutOfResourcesProps) {
+
+    return ReactDom.createPortal( 
+        <div className="popup-container">
+            <div className="overlay"></div>
+            <div className="popup"> 
+                <div className="popup-content">
+                            <h1 className="text-center">Oh No!</h1>
+                            <h2 className="fail">Level {level} Failed</h2>
+                            <h2 className="text-center"> Your player ran out of resources!</h2>
+                            <Link
```

Comment:
Maybe change this to "You ran out of resources!"

Reply:
I will make this change